brayns/common/scene/Scene.h

```diff
@@ -230,7 +230,7 @@ class Scene
     // Model
     PrimitivesMap _primitives;
     TrianglesMeshMap _trianglesMeshes;
-    Materials _materials;
+    MaterialsMap _materials;
```

Comment:
There was a mix between vectors and maps, but I decided to go for maps, mainly for access. I know that I will soon have to improve material management by being able to select then by name or ID. So maps are in the end more appropriate but I understand this is not clear in that code.